docs/resources/iam_policy.md

```diff
@@ -0,0 +1,312 @@
+---
+Title: "IAM Policy Resource"
+Description: |-
+    Creating the Tanzu Kubernetes IAM policy resource.
+---
+
+# IAM Policy
+
+The `tanzu-mission-control_iam_policy` resource enables you to add, update and delete role bindings to a particular scope for management through Tanzu Mission Control.
+
+IAM policy (also known as Access Management) allows you to implement role-based access control (RBAC) in Tanzu Mission Control. For more information, see [Access Control.][access-control]
+
+[access-control]: https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-concepts/GUID-EB9C6D83-1132-444F-8218-F264E43F25BD.html
+
+## Policy Scope and Inheritance
+
+In the Tanzu Mission Control resource hierarchy, there are five levels at which you can specify IAM policy resources:
+- **organization** - `organization` block under `scope` sub-resource
+- **object groups** - `cluster_group` block under `scope` sub-resource
```

Comment:
Can we only keep 3 levels and specify each of the allowed blocks under a particular level in the form of a list?